src/cosalib/qemuvariants.py

```diff
@@ -216,7 +220,7 @@ def image_meta(self):
 
     def set_platform(self):
         run_verbose(['/usr/lib/coreos-assembler/gf-platformid',
-                     self.image_qemu, self.tmp_image, self.platform])
+                     self.image_qemu, self.tmp_image, self.ignition_platform_identifier])
```

Comment:
@jlebon do you think something like this is too far retched? It allows me treating kubevirt as a separate platform (like creating the right sections in the release json), but should inject the desired platform id for ignition into the guest.

Reply:
Removed.